src/ConfigEditor.tsx

```diff
@@ -76,7 +75,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
               labelWidth={8}
               inputWidth={20}
               onChange={this.onURLChange}
-              value={jsonData.scalyrUrl}
+              value={jsonData.scalyrUrl || 'https://app.scalyr.com/'}
```

Comment:
LGTM, this was also my first suspicion when I quickly glanced over the git repo history and checking code which touched ``scalyrUrl`` related stuff - https://github.com/scalyr/scalyr-grafana-datasource-plugin/commit/4c548155e952cf75ce3fa541f00171fd3d7748e0#diff-1d0cc51dac0e69e79828e482663c8760447400c4cba9e4820ffb7f9f933312aeL78

Reply:
Once the fix is confirmed and we get a chance, it would be great if we could add a test case for it.